src/Ledger/Introduction.lagda

```diff
@@ -19,12 +19,38 @@ private variable
   sigs : List Sig
 \end{code}
 
-Repository: https://github.com/input-output-hk/formal-ledger-specifications
-
-This document describes the formalization of the Cardano ledger specification in the Agda programming
-language and proof assistant. The specification formalized here is that of the Conway era, described
-in detail in the Cardano Improvement Proposal (CIP) 1694,
-\href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694}{github.com/cardano-foundation/CIPs/CIP-1694}.
+This is the work-in-progress specification of the Cardano ledger. The
+current status of each individual era is described in Table \ref{fig:eras-progress}.
+
+\begin{longtable}[h!]{|l l l l|}
+\hline
+Era  & Figures & Prose & Cleanup \\
```

Comment:
I think we'll need to make a separate PDF just for the Conway spec at this point, because anything this incomplete probably shouldn't be a document to point people at. The point of this table is simply to make it clear where this document is currently at, so when people go looking for Babbage-era stuff they don't accidentally go look here.